src/patternfly/components/Spinner/spinner.scss

```diff
@@ -0,0 +1,252 @@
+// Spinner Component
+
+.pf-c-spinner {
+  --pf-c-spinner--AnimationDuration: 1.5s;
+  --pf-c-spinner--AnimationTimingFunction: cubic-bezier(.77, .005, .315, 1);
+
+  // set the overall width and height of the spinner to be consistent with the icon size
+  --pf-c-spinner--Width: var(--pf-global--icon--FontSize--xl);
+  --pf-c-spinner--Height: var(--pf-global--icon--FontSize--xl);
```

Comment:
I was in a constant internal struggle over all the shared variables. You'd do this then?
```  --pf-c-spinner--Width: var(--pf-global--icon--FontSize--xl);
  --pf-c-spinner--Height: var(--pf-c-spinner--Width);```

Reply:
I believe having Width and Height be declared separately by the same variable is better than basing Height off of Width.

Given height is based off of width, if the user wants to change width they'd also change the height as well without wanting to. But the same wouldn't happen the other way around if they changed height instead

If we want to keep the aspect ratio and have it linked, there's always the possibility of using Radius instead of Width and Height

Reply:
Yeah, good point. I personally like using radius or diameter but was trying to play nicely with our variable rules :) Anyone have a problem with defining radius or diameter and using that?

Reply:
For now, I have left it as Width and Height and added a comment noting that they need to be the same.